views/histogram.py

```diff
@@ -25,30 +27,32 @@ def get_layout(session_id):
     default_end_date = start_date + timedelta(weeks=1)
     filtered_data = eq_data.filter_by_dates(start_date, default_end_date)
 
-    default_size_column = eq_data.get_column_params(
-        eq_data.get_magnitudes().name
-    )
+    default_column = filtered_data.get_magnitudes()
+    default_nbins = 10
 
     return html.Div([
             dbc.Col(
                 html.Div(
                     id='histogram',
-                    children=histogram.get_component(None)
+                    className='plot_sidebar_open',
+                    children=histogram.get_component(
+                        default_column, default_nbins)
                 )
             ),
             dbc.Col(histogram_config.get_component(
-                eq_data.data.columns, start_date, end_date, default_end_date))
+                eq_data.data.columns, start_date, end_date,
+                default_end_date, default_column.name))
     ])
```

Comment:
Adding an `dbc.Row` element to encompass this sets the configurations next to the plot